www/md_base/src/app/common/services/buildbot/buildbot.service.coffee

```diff
@@ -0,0 +1,203 @@
+class BuildbotService extends Factory
```

Comment:
I would prefer that you symlink those files. As you have no intention to modify them for md_base, I would prefer to avoid duplication.